packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -276,7 +281,7 @@ export default class PciStoragesContainersService {
           angular.merge(
             {
               method: 'GET',
-              url,
+              url: url.url ? url.url : url,
```

Comment:
```suggestion
              url: url?.url ?? url,
```

Reply:
when i use this syntax i get an error SyntaxError: Optional chaining cannot appear in left-hand side

Reply:
as you weren't using optional chaining before it should work without it 
```suggestion
              url: url.url ?? url,
```